src/com/oltpbenchmark/schema/defaultschema/DefaultTable.java

```diff
@@ -0,0 +1,48 @@
+package com.oltpbenchmark.schema.defaultschema;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.oltpbenchmark.schema.Column;
+import com.oltpbenchmark.schema.Table;
+import com.oltpbenchmark.schema.TableSchema;
+
+public class DefaultTable extends Table {
+    
+    private final String tablespace;
+    
+    public DefaultTable(TableSchema schema) {
+        super(schema);
```

Comment:
It is possible to call another constructor via `this(schema, null)`

Reply:
Done